validator/sql_instance_type_test.rego

```diff
@@ -0,0 +1,158 @@
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GCPSQLInstanceTypeConstraintV1
+
+import data.validator.gcp.lib as lib
+
+#Importing the test data
+import data.test.fixtures.sql_instance_type.assets as fixture_assets
+
+# Importing the test constraints
+import data.test.fixtures.sql_instance_type.constraints.allow_all as fixture_constraint_allow_all
+import data.test.fixtures.sql_instance_type.constraints.deny_all as fixture_constraint_deny_all
+import data.test.fixtures.sql_instance_type.constraints.mysql as fixture_constraint_mysql
+import data.test.fixtures.sql_instance_type.constraints.postgres as fixture_constraint_postgres
+import data.test.fixtures.sql_instance_type.constraints.sql as fixture_constraint_sql
```

Comment:
Nit: This naming is ambiguous and requires cross-referencing to understand the tests. It might be easier to use the constraint name throughout - `sql_type_deny_mysql_and_sql`.

Reply:
done.